.github/workflows/publish.yml

```diff
@@ -0,0 +1,37 @@
+name: Publish
```

Comment:
If you don't mind, I would apply all comments from https://github.com/tarantool/smtp/pull/38 here.

Reply:
The idea about uploading .src.rock is nice. So implemented it for the smtp module in https://github.com/tarantool/smtp/pull/41 and I'll keep it here.

Reply:
Done.